scripts/get_latest_release_candidate.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Checks for the latest release candidate for the current version.
+
+Gets all releases from PyPI, filters for release candidates of the current streamlit version.
+If there is an existing RC, prints that RC version. If there is no RC, returns None.
+"""
+import json
+import os
+import requests
+
+
+def check_last_rc():
+    """Checks PyPI for existing release candidates"""
+
+    current_version = os.getenv("DESIRED_VERSION")
+
+    url = "https://pypi.org/pypi/streamlit/json"
+    response = requests.get(url).text
+    result = json.loads(response)
```

Comment:
nit: I think there's a `requests.json` convenience method that'll let you drop the `json` import